examples/cfq/train.py

```diff
@@ -338,6 +353,11 @@ def train_model(learning_rate: float = None,
     Do the training on data_source.train_dataset and evaluate on
     data_source.dev_dataset at each epoch and log the results
     """
+  if os.path.isdir(model_dir):
+    shutil.rmtree(model_dir)
+  os.makedirs(model_dir)
+  logging_file_name = os.path.join(model_dir, 'logged_examples.txt')
+  logging_file = open(logging_file_name,'w')
```

Comment:
This means the file will be opened during the entire time of training, which is not so nice and could go wrong if your training crashes. I would pass `model_dir` to `evaluate_model` below and do the file operations there.

Reply:
won't that impact the speed? opening and closing the file each time I am evaluating the model?

Reply:
No that is superfast, and keeping a file open like this is not good practice.